source/includes/administration/_billing_settings.md

```diff
@@ -0,0 +1,201 @@
+### Billing Settings
+
+<!-------------------- FIND BILLING SETTINGS -------------------->
+
+#### Find billing settings associated to an organization
+
+`GET /reseller/settings/billing/find?organizationId=:id`
+
+Retrieve the billing settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
+
+```shell
+# Retrieve the billing settings
+curl "https://cloudmc_endpoint/api/v1/reseller/settings/billing/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
```

Comment:
isnt supposed to be:
```suggestion
curl "https://cloudmc_endpoint/api/v1/rest/reseller/settings/billing/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
```

Reply:
I am not sure that is why I am asking, we usually use the `rest` path for the api but I might be wrong I havent tested this endpoint yet.

But if yes, then I think we may need to modify the ones below

Reply:
If not please disregard my comment

Reply:
yeah, the `v1` needs to be dropped in favour of `rest`

Reply:
done aa2d00770506d18f625dbda219bac3f6bf929b6b